driver/event_table.c

```diff
@@ -384,6 +384,9 @@ const struct ppm_event_info g_event_info[PPM_EVENT_MAX] = {
 	/* PPME_SYSCALL_BPF_2_X */{"bpf", EC_OTHER, EF_CREATES_FD, 1, { {"fd", PT_FD, PF_DEC} } },
 	/* PPME_SYSCALL_MLOCK2_E */{"mlock2", EC_MEMORY, EF_NONE, 0},
 	/* PPME_SYSCALL_MLOCK2_X */{"mlock2", EC_MEMORY, EF_NONE, 4, {{"res", PT_ERRNO, PF_DEC}, {"addr", PT_UINT64, PF_HEX}, {"len", PT_UINT64, PF_DEC}, {"flags", PT_UINT32, PF_HEX, mlockall_flags}}},
+	/* PPME_SYSCALL_FSCONFIG_E */{"fsconfig", EC_SYSTEM, EF_NONE, 0},
+	/* PPME_SYSCALL_FSCONFIG_X */{"fsconfig", EC_SYSTEM, EF_USES_FD | EF_WRITES_TO_FD, 7, {{"res", PT_ERRNO, PF_DEC}, {"fd", PT_FD, PF_DEC}, {"cmd", PT_ENUMFLAGS32, PF_DEC, fsconfig_cmds}, {"key", PT_CHARBUF, PF_NA}, {"value_bytebuf", PT_BYTEBUF, PF_NA}, {"value_charbuf", PT_CHARBUF, PF_NA}, {"aux", PT_INT32, PF_DEC}}},
```

Comment:
`EF_WRITES_TO_FD` do we need this for some filterchecks stuff?

Reply:
I think it's just because it describes what the syscall does :)

Reply:
If I understand well we are not reading or writing data from/to this file descriptor like `write`, `send`, `read`, `copy_file_range` WDYT?

Reply:
we are just configuring the file system :thinking:

Reply:
That's what we have for io_uring_enter too! We are actually writing to an FD even if there's no actual IO.
Everything depends on how you consider that, but the flag is called WRITES_TO_FD not HAS_WRITE_IO imho of course :)

Reply:
```
case TYPE_ISIO:
		{
			ppm_event_flags eflags = evt->get_info_flags();
			if(eflags & (EF_READS_FROM_FD | EF_WRITES_TO_FD))
			{
				m_u32val = 1;
			}
			else
			{
				m_u32val = 0;
			}
		}
```

Ok, you are right. I actually missed this one. We are not doing any real IO at all here; i guess i will remove the flag here and for `io_uring_enter` too (it is not a break because the flag is just used in a parser to determine whether fd is a tracer).